nixio/dimensions.py

```diff
@@ -315,21 +315,36 @@ def position_at(self, index):
         sample = self.sampling_interval
         return index * sample + offset
 
-    def index_of(self, position):
+    def index_of(self, position, leq=True):
         """
         Returns the index of a certain position in the dimension.
 
         :param position: The position.
+        :param leq: Less or Equal mode (default True).
+                    Whether to return the index if it matches the position exactly.
```

Comment:
not sure whether I really understand this first sentence. Maybe remove it? 
Not sure, if the `leq` flag alone is sufficient. For starting points of ranges we need the greater of equal.

Reply:
Oh, I read https://github.com/G-Node/nix/blob/master/docs/slicing_and_indices.md#rangematching-enumeration wrong.  Disregard the reply to the first comment then.  You're right, start positions should be `geq` and I made them `leq`.